File: sip-log/src/main/kotlin/com/basicfu/sip/log/TestController.java
package com.basicfu.sip.log;

import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

/**
 * @author basicfu
 * @date 2018/5/24
 */
@RestController
public class TestController {
    @GetMapping("/test")
    public void test(){
        System.out.println("123");
    }
}

File: sip-core/src/main/kotlin/com/basicfu/sip/core/TestController.kt
package com.basicfu.sip.core

import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController

/**
 * @author basicfu
 * @date 2018/6/22
 */
@RestController
@RequestMapping("/test")
class TestController{
    @GetMapping("/test")
    fun test(){

    }
}
